src/components/page/MembersPage.tsx

```diff
@@ -8,10 +10,24 @@ interface IMembersPageProps {
 export const MembersPage = async ({ id }: IMembersPageProps) => {
   const users: IUser[] = await fetchUserList(id);
   return (
-    <>
-      {users.map((user, i) => (
-        <div key={i}> {user.name} </div>
-      ))}
-    </>
+    <div className="px-4">
+      <div className="flex items-center mt-[35px] mb-5">
+        <h2 className="text-xl mr-44">Members</h2>
```

Comment:
### [Suggestion]
In line 15, you're using `mr-44` to create the space between title and Icons.
I can understand `44` **accurately reflects the design in Figma, it's actually great attitude!!**

However, even though it looks perfect in the window size of iPhone SE, **this margin causes unwanted space in some devices, like the one in the screenshot below...**

<img width="564" alt="Screenshot 2023-12-28 at 21 13 28" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99339182/6ad1fe0e-0f6b-4723-838a-1a2d92014f0c">

Reply:
I suppose it might be better to use `justify-between` to create the space between title and icons. 
In this way we can always set the title to left with 16px space from left edge of screen, and set the icons to right with the same space from right edge.
Plz take a look at how I implemented in my branch on your convenience if it's needed! 👍 

`  <div className="h-12 w-full flex items-center justify-between mt-[35px] mb-5">
        <h2 className="text-xl">Members</h2>
        <div className="flex justify-center items-center gap-0.5">
          <div className="w-12 h-12 flex justify-center items-center">
            <Icon icon={MenuMeatballIcon} className="w-5 h-1" />
          </div>
          <div className="w-12 h-12 flex items-center justify-center">
            <Icon icon={PlusIcon} className="w-[1.125rem] h-[1.125rem]" />
          </div>
        </div>
      </div>`

Reply:
HI, Kanta
Thank you for your review and comment! noted I need to consider responsiveness, appreciate it!
There is one thing I would like to know about in your code,
 `<div className="flex justify-center items-center gap-0.5">`
do I need to set `justify-center` , `items-center` and `gap-0.5` ??
I would appreciate if you kindly explain it.
Thanks!

Reply:
@ShoeheyOt 
Sorry,  I thought `justify-center items-center` was needed to centralize icons,  but found out now acutally it was unnecessary. You can remove them.

About `gap-0.5`, there's actually 2px gap between the icons in design, so it was needed to make that gap. It's really slight one tho, can you check it on figma, too?

<img width="449" alt="Screenshot 2023-12-28 at 23 14 38" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99339182/d4e3c73b-47e0-41b4-8d78-3c509072e6e8">

Reply:
Wow! I see the 2px gap on Figma! thank you for quick and clear explanation! I have already updated the code.
I will push after I get reply from Kota. 
Thanks!